shared/cloud/ess-getting-started.asciidoc

```diff
@@ -0,0 +1,25 @@
+[cloud-ess-setup]
+== Set up on Cloud
+
+// Need this at a different level? Use leveloffset: +1 to change to H3, 
+// for example, and then change the offset back with leveloffset: 0 afterwards.
```

Comment:
This is awesome! My comment from the weekly meeting: To keep these as flexible as possible, I'm not sure that we should include headers in these files. Instead, we could comment out a suggested header, which would encourage harmony across documentation, but allow each product to customize headings and anchor tags as necessary.

Reply:
A side effect of this is that we wouldn't be able to include links live you done at the bottom of this file: https://github.com/elastic/docs/pull/1815/files#diff-934228d98950ff95c8c53444966daeaeR25

Reply:
Another side effect is the "edit" button linking to the product documentation instead of the docs repo documentation. That might actually be a good thing though?

Reply:
I tinkered with an `ifeval::` statement to enable or disable the heading in your book, but in the end decided that your suggestion is simply the most straightforward ... and commented out the heading. I think keeping it as flexible and simple as possible is the right thing to do here.